azurelinuxagent/distro/default/env.py

```diff
@@ -94,6 +100,22 @@ def handle_dhclient_restart(self):
            self.distro.dhcp_handler.conf_routes()
            self.dhcpid = newpid
 
+     def handle_fstab_update(self):
```

Comment:
Please stick to 80-character wide lines. If you get notices about this, please run `pep8` command on the source code.